guide/inflation.rst

```diff
@@ -151,72 +251,143 @@ In the namelist each entry has two values. The first is for Prior inflation and
-"``ncap2``" on a copy of a restart file. Inflation mean and sd values look exactly like restart values, arranged by
-variable type like T, U, V, etc.
-
-Here's an example of using ncap2 to set the T,U and V inf values:
-
-.. container:: unix
-
-   ::
-
-        ncap2 -s 'T=1.0;U=1.0;V=1.0' wrfinput_d01 input_priorinf_mean_d01.nc
-        ncap2 -s 'T=0.6;U=0.6;V=0.6' wrfinput_d01 input_priorinf_sd_d01.nc
-        -or-
-        ncap2 -s 'T(:,:,:)=1.0;U(:,:,:)=1.0;V(:,:,:)=1.0' wrfinput_d01 input_priorinf_mean_d01.nc
-        ncap2 -s 'T(:,:,:)=0.6;U(:,:,:)=0.6;V(:,:,:)=0.6' wrfinput_d01 input_priorinf_sd_d01.nc
-
-Some versions of the NCO utilities change the full 3D arrays into a single scalar. If that's your result (check your
-output with ``ncdump -h``) use the alternate syntax or a more recent version of the NCO tools.
+it possible to restart an experiment. For many of the supported models, the scripting that
+does the renaming and copying of the inflation files may already be provided.
+
+.. Important::
+		Inflation is only applied to the variables that are **updated** by DART
+                by specifying ``UPDATE`` within the ``&model_nml`` section of ``input.nml``.
+		Alternatively, if the variables are not updated (``NO_COPY_BACK``) then inflation
+		will not impact them. ``NO_COPY_BACK`` variables are often used to compute the forward
+		operators, however, they don't take part in the update and as such
+		they will not be inflated.
+
+                Assigning a variable as ``NO_COPY_BACK`` is the **only way** to prevent inflation
+                from being applied to a variable when inflation is turned on.  If a variable is
+                not updated by observations because either 1) observation gaps exist in space or time 
+                2)the state is outside the spatial localization distance, or  3) variable localization 
+                was applied through the ``obs_impact_tool``, --the inflation update for that part of the
+                state will revert to the inflation damping method, and inflation will still be applied 
+                to all parts of the state space.
+
+                DART does not allow for variable specific ``inf_damping`` settings thus if an immediate
+                response time is required in the application of inflation, an alternative is to
+                customize the ``input.nml`` to switch between ``UPDATE`` and ``NO_COPY_BACK`` as needed. 
+                For an example discussing the impact of variable localization on inflation
+                see `issue 777. <https://github.com/NCAR/DART/issues/777>`_
+
+The suggested procedure for testing inflation:
+
+1. Start without any inflation; i.e., ``inf_flavor = 0, 0`` and assess the performance. For a healthy ensemble
+DA system, one expects the prior RMSE and the total spread to be of the same order. We often use the ensemble
```

Comment:
Shouldn't this say the RMSE and tot_spread should be roughly the same 'value' or 'magnitude'?  If you say they should be roughly the same 'order' this might suggest to me 'order of magnitude' which means even a 4-5x difference could be acceptable -- but I think we don't want the RMSE/tot_spread ratio being outside of 0.5 to 1.5 range.... correct?   Could you clarify this for me ?

Reply:
The term that is often used in the literature is 'order'. Overall, I think you want to avoid insanely large consistency values but sometimes 4-5x could actually be acceptable especially if it doesn't persist over long assimilation windows. So, I wouldn't be so stringent on the range of the ensemble consistency values. Clearly, a consistency of 1 is what we're after but this may not be even possible in systems where error growth is limited. Yet, I'm going to use the word 'magnitude' because it provides a better guidance for the users.